### PR TITLE
Style guide updates

### PR DESCRIPTION
**Pull request by anselld**
Hi Hansie. Please could you review the updated Style Guide. I included the requirement for UK punctuation and a reference, as well as Appendix B that will contain TLU terminology.
Thanks and regards
Denise

### CODE REVIEW

File: src/preface/style-guide.md [hunk 1]
```diff
@@ -48,7 +49,15 @@ TLU content is created in [Markdown](https://www.markdownguide.org/) format and
...
-Use the United States (US) spelling standard. The applicable dictionary is Merriam-Webster Online [[1]]. 
+As per the United States (US) spelling standard. The applicable dictionary is Merriam-Webster Online [[1]]. 
+
+### Quotation Marks
+
+As per the American style. Use double quotation marks for a first quotation and single quotation marks for a quotation within a quotation.
+
+### Punctuation
+
+As per the United Kingdom (UK) punctuation standard. Place commas and full stops as advised in [[4]].
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]:
```suggestion
As per the United Kingdom (UK) punctuation standard. Place commas and full stops outside the quotation marks as advised in [[4]]. 
```

File: src/preface/style-guide.md [hunk 2]
```diff
@@ -212,17 +192,15 @@ levels and content.
 
    - **Other headings as appropriate (as heading level 2 and lower)**
 
-      Structure the body of your report by using headings and subheadings. Ordering these headings logically helps you to 
-      present your information effectively. Headings make it easier for readers to find specific information.
+      Structure the body of your report by using headings and subheadings. Ordering these headings logically helps you to present your information effectively. Headings make it easier for readers to find specific information.
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]: Line break? See previous related comment.

File: src/preface/style-guide.md [hunk 3]
```diff
@@ -175,34 +163,26 @@ Where a list is not a sentence and does not complete a preceding part of a sente
...
 
-
-
 ### Case Formatting
 
-[Appendix A](#appendix-a-lower-case-words-used-in-title-case-formatting) contains a list of lower-case words used in 
-title case formatting in headings and captions (if used).
-
-
+[Appendix A](#appendix-a-lower-case-words-used-in-title-case-formatting) contains a list of lower-case words used in title case formatting in headings and captions (if used).
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]: Why removing the line break? This is confusing the review a bit. Suggest changing it back to the original so the style guide can be in line with the 120 character width suggestion.

File: src/preface/style-guide.md [hunk 4]
```diff
@@ -212,17 +192,15 @@ levels and content.
...
 
       - **Numbered Lists:** Use numbered lists when the order of the items in the list is important, such as in procedures.
 
       - **Bulleted Lists:** Use bulleted lists when the order of the items in the list is not important.
 
    - **Conclusions, Observations, Recommendations (as heading level 2)**
 
-      The conclusion complements the purpose of the report. It concisely summarizes the findings of the report and gives a 
-      future strategy, if required.
+      The conclusion complements the purpose of the report. It concisely summarizes the findings of the report and gives a future strategy, if required.
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]: Line break? See previous related comment.

File: src/preface/style-guide.md [hunk 5]
```diff
@@ -234,20 +212,16 @@ levels and content.
 
       - **Appendix A: Name (as heading level 3)**
 
-        Rather than inserting an entire supporting document into an appendix, provide a text reference and list the reference 
-        in the references section.
+        Rather than inserting an entire supporting document into an appendix, provide a text reference and list the reference in the references section.
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]: Line break? See previous related comment.

File: src/preface/style-guide.md [hunk 6]
```diff
@@ -234,20 +212,16 @@ levels and content.
...
 
       - **Appendix B: Name  (as heading level 3)**
 
-        If figure and table numbers are used in the report, the figure and table numbering in the appendices follows on from 
-        the figure and table numbers used in the report.
+        If figure and table numbers are used in the report, the figure and table numbering in the appendices follows on from the figure and table numbers used in the report.
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]: Line break? See previous related comment.

File: src/preface/style-guide.md [hunk 7]
```diff
@@ -298,23 +268,16 @@ before this list.
...
-  
-
 ### Figures and Tables
 
 The use of captions, as well as figure and table numbering, is optional. If you choose to use numbering and captions, 
 these guidelines will help to promote consistency in TLU layout:
 
-- Number figures and tables in each section sequentially, with the table caption above the table and the figure caption 
-below the figure.
+- Number figures and tables in each section sequentially, with the table caption above the table and the figure caption below the figure.
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]: Line break? See previous related comment.

File: src/preface/style-guide.md [hunk 8]
```diff
@@ -298,23 +268,16 @@ before this list.
...
 - Type figure and table captions in upper and lower-case (title case).
 - Type Figure x: or Table X: before the caption, as applicable.
 - Center figures and tables on the page.
-- Place figures and tables as soon as possible after they are first referred to in the text. The text reference, if figure 
-and table numbering is not used, would then be "the following figure..." or "the following table...." This helps to avoid 
-confusion.
-
-
+- Place figures and tables as soon as possible after they are first referred to in the text. The text reference, if figure and table numbering is not used, would then be "the following figure..." or "the following table...". This helps to avoid confusion.
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]: Line breaks? See previous related comment.

File: src/preface/style-guide.md [hunk 9]
```diff
@@ -421,29 +383,23 @@ A **reference link** has two parts. The first part of a reference link has two s
...
 [[1]] Merriam-Webster Online Dictionary [online]. 
-Available: https://www.merriam-webster.com/. Date accessed: 2008-02-01.
+Available: https://www.merriam-webster.com/. Date accessed: 2019-02-01.
 
 [1]: https://www.merriam-webster.com
 "Merriam-Webster Online Dictionary"
 
-The full online reference is inserted after `[[1]]`; and the pop-up text link (which can be seen by hovering your cursor 
-over the text reference in [Spelling](#spelling)) is inserted after `[1]:`. 
+The full online reference is inserted after `[[1]]`; and the pop-up text link (which can be seen by hovering your cursor over the text reference in [Spelling](#spelling)) is inserted after `[1]:`.
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]: Line break? See previous related comment.

File: src/preface/style-guide.md [hunk 10]
```diff
@@ -421,29 +383,23 @@ A **reference link** has two parts. The first part of a reference link has two s
...
 
-
-
 ### List of Contributors
 
-The contributors are listed in a bulleted list via their GitHub account URLs. The author is listed first, followed by any 
-reviewers or people who contributed via pull requests. Refer to [Contributors](#contributors) for an example.
-
-
+The contributors are listed in a bulleted list via their GitHub account URLs. The author is listed first, followed by any reviewers or people who contributed via pull requests. Refer to [Contributors](#contributors) for an example.
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]: Line break? See previous related comment.

File: src/preface/style-guide.md [hunk 11]
```diff
@@ -465,6 +421,13 @@ Publication 811 - 2008 Edition [online]. Available: https://physics.nist.gov/cuu
 [3]: https://physics.nist.gov/cuu/pdf/sp811.pdf
 "Guide for the Use of the International System of Units (SI)"
 
+[[4]] The Oxford Guide to Style [online]. 
+Available: http://www.eng-lang.co.uk/ogs.htm. Date accessed: 2019-02-04.
+
+[4]: https://www.merriam-webster.com
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]:
```suggestion
[4]: http://www.eng-lang.co.uk/ogs.htm
```

File: src/preface/style-guide.md [hunk 12]
```diff
@@ -465,6 +421,13 @@ Publication 811 - 2008 Edition [online]. Available: https://physics.nist.gov/cuu
...
+
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]:
```suggestion
```

File: src/preface/style-guide.md [hunk 13]
```diff
@@ -465,6 +421,13 @@ Publication 811 - 2008 Edition [online]. Available: https://physics.nist.gov/cuu
...
+"http://www.eng-lang.co.uk/ogs.htm"
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]:
```suggestion
"The Oxford Guide to Style"
```

File: src/preface/style-guide.md [hunk 14]
```diff
@@ -474,42 +437,87 @@ Publication 811 - 2008 Edition [online]. Available: https://physics.nist.gov/cuu
...
+
+
+
+
+
+### Appendix B: Tari Labs University Terminology Conventions
+
+**Note:** This list is a work in progress.
+
+| Term         | Term | Term | Term |
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]: Please resize this table to only allow for the current content.

File: src/preface/style-guide.md [hunk 15]
```diff
@@ -474,42 +437,87 @@ Publication 811 - 2008 Edition [online]. Available: https://physics.nist.gov/cuu
...
+| consists     | is        | their     | within     |
+| does         | it        | them      | without    |
+| down         | its       | then      | worst      |
+| e.g.         | least     | there     |            |
+
+
+
+
+
+### Appendix B: Tari Labs University Terminology Conventions
```
Comment (hansieodendaal) [inline src/preface/style-guide.md]: Please add a paragraph/section after 'Case Formatting' to explain Appendix B. See example below.

### Terminology

Words change over time, and new concepts are formed daily. Suggested terminology for use in TLU reports is listed in Appendix B.

File: src/preface/style-guide.md (hunk 3 above)
Comment (anselld) [reply src/preface/style-guide.md]: Thanks. Will apply.

File: src/preface/style-guide.md (hunk 5 above)
Comment (anselld) [reply src/preface/style-guide.md]: Thanks.

File: src/preface/style-guide.md (hunk 6 above)
Comment (anselld) [reply src/preface/style-guide.md]: Thanks.

File: src/preface/style-guide.md (hunk 7 above)
Comment (anselld) [reply src/preface/style-guide.md]: Thanks.

File: src/preface/style-guide.md (hunk 8 above)
Comment (anselld) [reply src/preface/style-guide.md]: Thanks.

File: src/preface/style-guide.md (hunk 9 above)
Comment (anselld) [reply src/preface/style-guide.md]: Thanks.

File: src/preface/style-guide.md (hunk 15 above)
Comment (anselld) [reply src/preface/style-guide.md]: Thanks - have done so.

File: src/preface/style-guide.md (hunk 14 above)
Comment (anselld) [reply src/preface/style-guide.md]: Thanks - have done so, but the table doesn't render correctly. Have I done something wrong?

**Comment by anselld** [comment]
Thanks for your review, Hansie. I've implemented your comments and will now commit and push the updated file. Please advise on the rendering of the table in Appendix B if you have a chance.

Thanks and regards
Denise

**Comment by anselld** [comment]
Hi Hansie
Thanks very much for your help and input. I've committed the updated version.
Regards
Denise

**Pull request merged**